### DISCUSSION TITLE
Schedule trigger in yaml

### DISCUSSION THREAD

**Discussion by Reedyuk** [codemagic-ci-cd/codemagic-docs#1768] [Ideas]
### Use case

```Text
I would like to schedule a nightly build, but i would like to have this set in the yaml file and not in the web console
```

### Idea

```Text
Have a new field for yaml that supports cron format to trigger a build
```

**Comment by Ycoi** [comment]
Thanks for the feedback @Reedyuk 

Is there some particular reason why you want to manage the schedule from the yaml instead of the web UI. Do you for example need to adjust the schedule often or do you need different schedules for different branches or something else? So can you maybe elaborate your use case more so that I get a better picture on why this would be a useful feature?

**Reply by Reedyuk** [reply]
IMO - all config should be driven through the yaml file, this in itself then has an audit history and commit notes.
It also means that if we were to use that script for another project, it would be copied over.

**Reply by Reedyuk** [reply]
the frequency doesnt change often, but then you could apply that principle to almost any config in the yaml file.

**Reply by Ycoi** [reply]
Thanks for the clarification! We’ll discuss this in the team and see what we can do.

**Reply by Ycoi** [reply]
Sorry that I only now get back to this. We are planning some bigger yaml related changes some time in the future and this makes sense to add as a part of those changes. So I marked this down as part of those changes but I can’t unfortunately give an estimate when this could be added.

**Comment by Ycoi** [comment]
I’ll close this feature request since unfortunately we don’t currently have plans to add this functionality. Our engineering team is  focusing on other improvements at the moment. 

Thanks for the request anyway, and if you have any other feature ideas in mind don’t hesitate to post them as new topics.
